satpy/readers/fci_l1c_nc.py

```diff
@@ -29,7 +29,8 @@
 
 .. note::
     This reader currently supports Full Disk High Spectral Resolution Imagery
-    (FDHSI) and High Spatial Resolution Fast Imagery (HRFI) data in full-disc ("FD") scanning mode.
+    (FDHSI) ,High Spatial Resolution Fast Imagery (HRFI) data in full-disc ("FD") scanning mode.
+    The african case ("AF") scanning mode has been added.
```

Comment:
```suggestion
    (FDHSI) and High Spatial Resolution Fast Imagery (HRFI) data in full-disc ("FD") scanning mode.
    In addition, it also supports the L1c format for the African dissemination ("AF"), where each file contains the 
    (masked) full-disc of a single channel.
```

Reply:
Please also add an hyperlink to the AfricaPUG here: https://www-cdn.eumetsat.int/files/2022-07/MTG%20EUMETCast%20Africa%20Product%20User%20Guide%20%5BAfricaPUG%5D_v2E.pdf

Reply:
It's done